rpc/src/v1/types/pubsub.rs

```diff
@@ -37,14 +37,8 @@ pub enum Result {
 #[derive(Debug, Serialize, Eq, PartialEq, Clone)]
 #[serde(rename_all="camelCase")]
 pub struct PubSubSyncStatus {
```

Comment:
I suppose this should be a simple `true/false` instead of `{"isSyncing": true/false}`

Reply:
leaving it as a struct allows us add more fields along the line.

Reply:
but is not compatible with go ethereum's output. The spec says:
> the result can either be a boolean indicating that the synchronization has started (true), finished (false) or an object with various progress indicators.

But the example shows:
```
"result":{"syncing":true,"status":{"startingBlock":674427,"currentBlock":67400,"highestBlock":674432,"pulledStates":0,"knownStates":0}}
```

So either let's rename the field to `syncing` (from `isSyncing`) and add an empty `status`(?) or let's just use `true/false` directly